types/lodash/common/common.d.ts

```diff
@@ -127,34 +127,29 @@ declare module "../index" {
          * @param key The key of the value to remove.
          * @return Returns `true` if the entry was removed successfully, else `false`.
          */
-        delete(key: string): boolean;
+        delete(key: any): boolean;
 
         /**
          * Gets the cached value for `key`.
          * @param key The key of the value to get.
          * @return Returns the cached value.
          */
-        get(key: string): any;
+        get(key: any): any;
 
         /**
          * Checks if a cached value for `key` exists.
          * @param key The key of the entry to check.
          * @return Returns `true` if an entry for `key` exists, else `false`.
          */
-        has(key: string): boolean;
+        has(key: any): boolean;
 
         /**
          * Sets `value` to `key` of the cache.
          * @param key The key of the value to cache.
          * @param value The value to cache.
          * @return Returns the cache object.
          */
-        set(key: string, value: any): Dictionary<any>;
-
-        /**
-         * Removes all key-value entries from the map.
-         */
-        clear(): void;
```

Comment:
I agree with the previous conversation about making this an optional parameter instead of removing it. The default cache provided by lodash has a `clear` method, so the types should represent that particular possibility.